_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-slate
```

Comment:
We don't need it

Reply:
Yah that was added by github. I will clean it.